samples/matter/template/src/app_task.h

```diff
@@ -18,11 +18,9 @@
 #endif
 
 struct k_timer;
-class AppFabricTableDelegate;
```

Comment:
Why we remove it only for template sample? And why we remove only code from app_task.h and not from app_task.cpp?